src/net/socket/tcp_sock.c

```diff
@@ -67,7 +67,7 @@ static int tcp_init(struct sock *sk) {
 	tcp_sk = to_tcp_sock(sk);
 	assert(tcp_sk != NULL);
 
-	debug_print(3, "tcp_init: sk %p\n", to_sock(tcp_sk));
+	log_debug("tcp_init: sk %p\n", to_sock(tcp_sk));
```

Comment:
What about this function? Shouldn't it be `log_debug("sk %p\n", to_sock(tcp_sk));`